config/components/manager/manager.yaml

```diff
@@ -61,10 +61,10 @@ spec:
         # TODO(user): For common cases that do not require escalating privileges
         # it is recommended to ensure that all your Pods/Containers are restrictive.
         # More info: https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted
-        # Please uncomment the following code if your project does NOT have to work on old Kubernetes
+        # Please comment the following code if your project has to work on old Kubernetes
```

Comment:
Maybe we should update these docs.

1.19 is so far out of support..

Reply:
Do you prefer to remove this comment all together ?

Reply:
I'm not sure if we even support that old of Kubernetes anymore.

So yes.